packages/pds/src/db/migrations/20231011T155513453Z-takedown-id-as-str.ts

```diff
@@ -0,0 +1,47 @@
+import { Kysely, sql } from 'kysely'
+import { Dialect } from '..'
+
+export async function up(db: Kysely<unknown>, dialect: Dialect): Promise<void> {
+  if (dialect === 'pg') {
+    await sql`
+      alter table "repo_root" alter column "takedownId" type varchar;
+      alter table "repo_blob" alter column "takedownId" type varchar;
+      alter table "record" alter column "takedownId" type varchar;
```

Comment:
I think for the rollout we'll need a slightly different approach, but shouldn't be too, too bad.

Reply:
yeah we can chat through what exactly this should look like 🤔